engine/release-notes/23.0.md

```diff
@@ -41,6 +41,28 @@ Changing the version format is a stepping-stone towards Go module compatibility,
 but the repository doesn't yet use Go modules, and still requires using a "+incompatible" version.
 Work continues towards Go module compatibility in a future release.
 
+## 23.0.5
+
+{% include release-date.html date="2023-04-26" %}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 23.0.5 milestone](https://github.com/docker/cli/milestone/79?closed=1)
+- [moby/moby, 23.0.5 milestone](https://github.com/moby/moby/milestone/118?closed=1)
+
+### Bug fixes and enhancements
+
+- Add the --all / -a option when pruning volumes. [docker/cli#4229](https://github.com/docker/cli/pull/4229)
+- Add --format=json for `docker info`. [docker/cli#4320](https://github.com/docker/cli/pull/4230)
```

Comment:
Probably could use backticks around the flags, and the full-stop to be moved to the end of the whole sentence (after the link)

```suggestion
- Add the `--all` / `-a` option when pruning volumes [docker/cli#4229](https://github.com/docker/cli/pull/4229).
- Add `--format=json` for `docker info` [docker/cli#4320](https://github.com/docker/cli/pull/4230).
```

Reply:
+1 for backticks

Reply:
Fixed!